### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by akbarsazish**
1- The form has already been added.
2-  Validate the input fields with (maxlength, required, type (email), and add placeholder to all)

### CODE REVIEW

File: pages/index.html [hunk 1]
```diff
@@ -163,18 +163,18 @@ <h2 class="lets-connect"> LET'S CONNECT </h2>
         <div class="contact-me">
             <div class="contact-info">
                 <h2 class="about-me-headline"> Contact me </h2>
-                <p class="contact-text"> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                <p class="contact-text"> If you have an application you are interested in <br> developing, a feature that you need built or a project <br> that needs coding. 
                     I’d love to help with it!</p>
                 <div class="form-container">
-                    <form>
+                    <form action="#">
```
Comment (levy002) [inline pages/index.html:169]: - [ ] Kindly make sure your `form` uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).


    - action value should be similar to https://formspree.io/f/{form_id}. Kindly make sure to create an account of the [formspree service](https://formspree.io/html) which will help you to see the message sent by users.
    - `method` value equals `POST`

File: pages/index.html [hunk 2]
```diff
@@ -163,18 +163,18 @@ <h2 class="lets-connect"> LET'S CONNECT </h2>
...
                         </div>
                         <div class="form-field">
-                        <input type="email" id="email" name="email" placeholder="loremipsum@gmail.com" required>
+                        <input type="email" id="email" name="email" placeholder="example@gmail.com" required>
                         </div>
                         <div class="form-field">
-                        <textarea id="message" name="message" placeholder="Write your message here..." rows="6" required></textarea>
+                        <textarea id="message" maxlength="500" name="message" placeholder="Write your message here..." rows="6" required></textarea>
                         </div>
                         <button type="button" class="see-project git get-intouch btnHover btnPressed btnDisabled"> Get in touch </button>
```
Comment (levy002) [inline pages/index.html:179]: - [ ] KIndly make sure your `Get in touch` button `type` attribute has the value `submit` which will help you to be able to submit the form successfully.

File: pages/index.html [hunk 3]
```diff
@@ -163,18 +163,18 @@ <h2 class="lets-connect"> LET'S CONNECT </h2>
         <div class="contact-me">
             <div class="contact-info">
                 <h2 class="about-me-headline"> Contact me </h2>
-                <p class="contact-text"> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                <p class="contact-text"> If you have an application you are interested in <br> developing, a feature that you need built or a project <br> that needs coding. 
                     I’d love to help with it!</p>
                 <div class="form-container">
```
Comment (levy002) [inline pages/index.html:168]: - [ ] Kindly make sure to align the `Contact me` title correctly and make sure to adjust the typography `(font-weight, font-family, font-size)` in order to match the provided Figma design.

- [ ] KIndly make sure to align correctly the contact section paragraph correctly and make sure to adjust the typography `(font-weight, font-family, font-size)`  in order to match the provided Figma design.

     - ### YOUR DESIGN
     
    image

     - ### FIGMA DESIGN
     
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @akbarsazish & @AztkMarty  , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._
`
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 👋 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors. Please rectify these linter errors and resubmit for review. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by frank1738 — APPROVED** [review]
Hi Team , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Form submission works correctly👍 


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — COMMENTED** [review]
Hi @akbarsazish, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses a pull request that has already been approved. Kindly note that you always have to create a `new branch` for each `milestone``(1 branch per milestone)`. For your project to be reviewed please create a new branch and implement the project requirements on that new branch and submit it for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**